MetaMask/metamask-extension
# Bridge tokens by redirecting to Portfolio from wallet overview page

## Explanation

This pull request adds a Bridge button to the wallet overview that opens the [MetaMask Portfolio Dapp bridge experience](https://portfolio.metamask.io/bridge) in a new tab when clicked.

## Changes
* Add new Bridge button and icon to eth-overview component
* Move Portfolio button next to displayed balance
* Add new `renderBackgroundColor` prop to IconButton to style Portfolio IconButton

## Screenshots/Screencaps

### Before

<img width="361" alt="Screenshot 2023-03-01 at 3 19 05 PM" src="https://user-images.githubusercontent.com/100321200/222288923-d7f9b159-bdb7-4c83-879f-29d039b6c3f7.png">


### After

<img width="361" alt="Screenshot 2023-03-03 at 10 51 31 AM" src="https://user-images.githubusercontent.com/100321200/222793149-d96c086a-352b-47c8-8840-6434651ef80f.png">





https://user-images.githubusercontent.com/100321200/222308227-981ecd35-e876-4754-b0ab-5bf793afd764.mov




## Manual Testing Steps
1. Open the extension
2. Click on the Bridge button
3. Verify that you are redirected to the MetaMask Portfolio Dapp bridge page
4. Open the extension
5. Click on the Portfolio button
6. Verify that you are redirected to the MetaMask Portfolio Dapp home page

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
